mobify/platform-scaffold
# Use <Link> from SDK

Use <Link> from the SDK

 **JIRA**: https://mobify.atlassian.net/browse/AMP-63

## Changes
- Use SDK test-utils
- Use SDK ampSDK
- Use <Link> from SDK

## Todos
- [x] Remove scaffold ampSDK and use SDK one
- [x] Link to published SDK version when it is available

## How to test-drive this PR
- Run `npm run dev`
- Check out http://localhost:3000/potions.html and http://localhost:3000/eye-of-newt.html
- Make sure tests pass
